@here/harp-examples/src/datasource_satellite-tile.ts

```diff
@@ -33,7 +33,8 @@ export namespace SatelliteDataSourceExample {
 
         const map = new MapView({
             canvas,
-            theme: "resources/berlin_tilezen_base_globe.json"
+            theme: "resources/berlin_tilezen_base_globe.json",
+            enableMixedLod: true
```

Comment:
I wonder if we should enable mixed LOD by default if `projection===sphereProjection`

Reply:
For this example it wouldn't work as it starts with `mercartorProjection` and can be switched to `sphereProjection`. The `enableMixedLod` would need to be changed every time `projection` is changed + some kind of override to allow to change it explicitly.